backend/src/test/java/com/redhat/cloud/notifications/routers/sources/SecretUtilsTest.java

```diff
@@ -528,10 +573,11 @@ void updateSecretsForEndpointCreateTest() {
         // Assert the results.
         Assertions.assertEquals(BASIC_AUTH_SOURCES_ID, props.getBasicAuthenticationSourcesId(), "the ID of the basic authentication secret from Sources doesn't match");
         Assertions.assertEquals(SECRET_TOKEN_SOURCES_ID, props.getSecretTokenSourcesId(), "the ID of the secret token's secret from Sources doesn't match");
+        Assertions.assertEquals(BEARER_TOKEN_SOURCES_ID, props.getBearerAuthenticationSourcesId(), "the ID of the bearer secret from Sources doesn't match");
 
         // Assert that the underlying function was called exactly two times, since we are expecting that both the
```

Comment:
```diff
-// Assert that the underlying function was called exactly two times, since we are expecting that both the
+// Assert that the underlying function was called exactly three times, since we are expecting that both the
```